runtime/major_gc.c

```diff
@@ -251,8 +251,7 @@ Caml_inline void pb_fill_mode(prefetch_buffer_t *pb)
 
 Caml_inline void pb_push(prefetch_buffer_t* pb, value v)
 {
-  CAMLassert(Is_block(v));
-  CAMLassert(!Is_young(v));
+  CAMLassert(Is_block(v) && !Is_young(v));
```

Comment:
This undoes a deliberate change done a few months ago by @MisterDA to split asserts so that, should one fire, we can know exactly which condition has triggered the assert, rather than having to guess.

Please consider reverting that particular change.

Reply:
Ah, good spot. I must have missed that in a rebase. Will fix.

Reply:
Should be done in https://github.com/ocaml/ocaml/pull/13616/commits/912f3665b7ced848a62a616203b3c0c9d18db391